xbmc/FileItem.cpp

```diff
@@ -490,8 +476,7 @@ void CFileItem::Reset()
   m_epgInfoTag.reset();
   delete m_pvrChannelInfoTag;
   m_pvrChannelInfoTag=NULL;
-  delete m_pvrRecordingInfoTag;
-  m_pvrRecordingInfoTag=NULL;
+  m_pvrRecordingInfoTag.reset();
```

Comment:
it got overlooked and needs to be changed too.

the problem is that we now no longer clear a copy of a tag, but clear the data in the reference to a tag.
fileitems are passed around all around the place, and Reset() can be called from about everything for whatever reason. so e.g. if we have some tag referenced in here, passed to a gui control which calls Reset() for whatever reason, we clear the tag in CPVRRecordings, instead of just the fileitem copy.